chain/chain/src/blocks_delay_tracker.rs

```diff
@@ -252,7 +285,19 @@ impl BlocksDelayTracker {
     }
 
     pub fn record_optimistic_block_ready(&mut self, block_height: BlockHeight) {
```

Comment:
Is it possible that we get incorrect metrics because of two blocks at the same height in case of a fork?

Reply:
I can imagine such scenario, but two blocks at same height is a malicious behaviour, so it should be too rare to care about in metrics.